config/samples/backends/edge/edge.yaml

```diff
@@ -0,0 +1,83 @@
+# Sample using Ceph as a glance backend
+# Requires a running Ceph cluster and its `/etc/ceph` files in secret `ceph-conf-files`
+# This can be achieved with the `ceph` target of `install_yamls`
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  glance:
+    template:
+      databaseInstance: openstack
+      databaseAccount: glance
+      secret: osp-secret
+      storageClass: ""
+      storageRequest: 10G
+      keystoneEndpoint: az0
```

Comment:
Can we change this to `default` so that the initial deployment can just use the defaults?